### PR TITLE
Remove partial subsidization

### LINKED ISSUE #2785: Remove Partial Account Subsidization
For simplicity we are removing partial subsidization for new accounts.

### PR DESCRIPTION
**Pull request by mvandeberg**
Closes #2785

### CODE REVIEW

**Comment by theoreticalbts** [comment]
Looks reasonable.

File: libraries/chain/steem_evaluator.cpp [hunk 1]
```diff
@@ -2432,13 +2384,20 @@ void claim_account_evaluator::do_apply( const claim_account_operation& o )
...
       _db.modify( gpo, [&]( dynamic_global_property_object& gpo )
       {
-         gpo.available_account_subsidies -= current_subsidy;
+         gpo.available_account_subsidies -= STEEM_ACCOUNT_SUBSIDY_PRECISION;
       });
    }
+   else
+   {
+      FC_ASSERT( o.fee == wso.median_props.account_creation_fee,
+         "Cannot pay more than account creation fee. paid: ${p} fee: ${f}",
```
Comment (plink01001) [inline libraries/chain/steem_evaluator.cpp]: Can this error condition also be hit if the paid fee is less than the account creation fee?

Comment (mvandeberg) [reply libraries/chain/steem_evaluator.cpp]: Yes,

```
if( o.fee.amount == 0 )
{
 ...
}
else
{
 FC_ASSERT( ... )
}
```

Comment (mvandeberg) [reply libraries/chain/steem_evaluator.cpp]: And there is a unit test for the case of the op fee being between 0 and the account creation fee.

**Pull request merged**